doc/api/tracing.md

```diff
@@ -33,8 +37,6 @@ The available categories are:
   `runInNewContext()`, `runInContext()`, and `runInThisContext()` methods.
 * `v8` - The [V8] events are GC, compiling, and execution related.
 
-By default the `node`, `node.async_hooks`, and `v8` categories are enabled.
```

Comment:
Is that not true anymore?

Reply:
It's misleading. Those categories are enabled by default only if tracing is turned on without any categories specified. Elsewhere in the doc, where we document how to turn those on, we say what the defaults are